packages/webdriverio/src/commands/browser/$$.js

```diff
@@ -42,14 +42,15 @@
  */
 import { webdriverMonad } from 'webdriver'
 import { wrapCommand } from '@wdio/config'
+import merge from 'lodash.merge'
 
 import { findElements, getPrototype as getWDIOPrototype, getElementFromResponse } from '../../utils'
 import { elementErrorHandler } from '../../middlewares'
 import { ELEMENT_KEY } from '../../constants'
 
 export default async function $$ (selector) {
     const res = await findElements.call(this, selector)
-    const prototype = Object.assign({}, this.__propertiesObject__, getWDIOPrototype('element'), { scope: 'element' })
+    const prototype = merge({}, this.__propertiesObject__, getWDIOPrototype('element'), { scope: 'element' })
```

Comment:
with that we don't need the first param anymore

Reply:
_merge() still requires the result object to be the initial param.